c10/core/impl/SizesAndStrides.h

```diff
@@ -297,7 +297,7 @@ class C10_API SizesAndStrides {
     memcpy(outOfLineStorage_, rhs.outOfLineStorage_, storageBytes(rhs.size_));
```

Comment:
```suggestion
    std::memcpy(outOfLineStorage_, rhs.outOfLineStorage_, storageBytes(rhs.size_));
```
while we are here.

Reply:
Actually, we should probably do the whole file here if we do it, feel free to disregard or do a separate PR.

Reply:
whole file?